### PR TITLE
fix: ui-resizable-autohide not removed on hover

### PR DESCRIPTION
**Pull request by dfenerski**
See #2106 

- [ ] Created tests which fail without the change (if possible)
- [ ] All tests passing (`yarn test`)
- [ ] Extended the README / documentation, if necessary

### CODE REVIEW

File: src/dd-resizable.ts [hunk 1]
```diff
@@ -130,7 +130,7 @@ export class DDResizable extends DDBaseImplement implements HTMLElementExtendOpt
   protected _mouseOver(e: Event): void {
     // console.log(`${count++} pre-enter ${(this.el as GridItemHTMLElement).gridstackNode._id}`)
     // already over a child, ignore. Ideally we just call e.stopPropagation() but see https://github.com/gridstack/gridstack.js/issues/2018
-    if (DDManager.overResizeElement || DDManager.dragElement) return;
+    if (DDManager.overResizeElement === this || DDManager.dragElement) return;
```
Comment (adumesny) [inline src/dd-resizable.ts:133]: yeah that is not the right fix - if we were already resizing, we should get a mouseOver, and if something else, we shouldn't make us the resizing widget.
the real fix is to figure why DDManager.overResizeElement is not being reset when dragging items fast or something...

Comment (dfenerski) [reply src/dd-resizable.ts:133]: Investigating this further showed that the problem lies with multiple instances of GridStack having the same element refs led to the problem. As this shouldn't be happening the problem really lies in the application code. 

Closing this and sorry for the disturbance.

BR

Comment (adumesny) [reply src/dd-resizable.ts:133]: | multiple instances of GridStack having the same element refs 

that shouldn't happen naturally, so there might still be a bug somewhere in the lib ? would be good to track where the issue lies.

Comment (dfenerski) [reply src/dd-resizable.ts:133]: The application was responsible for managing the instances, so we solved the issue by tweaking the logic there. I doubt it is a legit use case to create multiple instances with the *same* items so this one I guess is not in the lib.

Comment (kanikab15) [reply src/dd-resizable.ts:133]: Thank you @dfenerski. I faced the same issue and doing a grid.destroy() at the right place solved it.

**Pull request closed without merging**